joplin/base/models.py

```diff
@@ -65,24 +63,24 @@ class ServicePage(Page):
             ('collection_schedule_block', custom_blocks.CollectionScheduleBlock()),
             ('recollect_block', custom_blocks.RecollectBlock()),
         ],
-        verbose_name='Add any forms, maps, apps, or content that will help the resident use the service',
+        verbose_name='Add any maps or apps that will help the resident use the service',
     )
     additional_content = RichTextField(features=WYSIWYG_FEATURES, verbose_name='Write any additional content describing the service', blank=True)
     topic = models.ForeignKey(
         'base.Topic',
         on_delete=models.PROTECT,
         related_name='services',
     )
-    image = models.ForeignKey(TranslatedImage, null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
+    image = models.ForeignKey(TranslatedImage, null=True, blank=True, on_delete=models.SET_NULL, related_name='+', verbose_name='Choose an image for the service banner')
```

Comment:
Should we give any suggestions here about aspect ratio or image sizing?

Reply:
I used what's in the mockup. Adding help text for sizing makes sense. Should we make an issue?

It would be nice also if we could just make things work with aspect ratios or throw an error if it won't work, rather than typing out all the instructions.